backend/CircleForms/ModelLayer/Error.cs

```diff
@@ -53,9 +49,14 @@ public static Result<T> NotFound(string id)
         return new Result<T>(HttpStatusCode.NotFound, $"Entity {id} is not found");
     }
 
-    public Result<object> Error()
+    public static Result<T> Forbidden()
+    {
+        return new Result<T>(HttpStatusCode.Forbidden, "You're not allowed to access that resource");
+    }
+
+    public Error Error()

```

Comment:
`static`

Reply:
the point is to change result to the error

```cs
Error Validate()
{
    Result<T> result = GetResult();
    if (result.IsError) return result.Error();
}
```

In my opinion it looks better than

```cs
Error Validate()
{
    Result<T> result = GetResult();
    if (result.IsError) Result<T>.Error(result)
}
```